### PR TITLE
Adding binary authorization flow to application project pipelines

### PR DESCRIPTION
**Pull request by mike-ensor**
Before:  Binary Authorization is installed and the default policy is "DryRun" mode.  Three attestors are already created along with their asymmetric keys and all functionality to create Attestations.

After:  Attestations will be created by their respective attestors at the "build" (image creation), "quality" (staging environment signoff), "security" (post first prod, before second prod push).

Only the Java template has the new attestations added. NO change to the policy has been made, Dry Run / logging is still used.

### CODE REVIEW

**Comment by mike-ensor** [comment]
Will squash & merge when "LGTM" approval.

File: 1_clusters/binary-auth.tf [hunk 1]
```diff
@@ -30,6 +30,30 @@ resource "google_kms_key_ring" "keyring" {
   depends_on = [null_resource.resource-to-wait-on]
 }
 
+# Create a Google Secret containing the keyring name
```
Comment (morgante) [inline 1_clusters/binary-auth.tf:33]: Does this really need to be stored as a secret? I'm a little unsure on the intent here.

Comment (morgante) [reply 1_clusters/binary-auth.tf:33]: Got it, thanks for explaining.

Comment (mike-ensor) [reply 1_clusters/binary-auth.tf:33]: Is this a LGTM statement? Just need someone to verily things look good

Comment (morgante) [reply 1_clusters/binary-auth.tf:33]: Yeah, LGTM.

**Review by morgante — APPROVED** [review]

**Pull request merged**